packages/airnode-adapter/e2e/index.ts

```diff
@@ -38,6 +38,10 @@ const apiResponse = {
   bytes: '0x72616e646f6d20737472696e6700000000000000000000000000000000000000',
   big: {
     decimal: '11223344556677889900',
+    // The maximum number that fits in int256
```

Comment:
> I think it'd be great to include a test specifically for #752 here.

@dcroote I added these tests. What do you think?

For some reason using the powers (`2 ** 255`, `2 ** 255 - 1`) results in a `Error: invalid BigNumber string`, so I included them as strings instead.